src/koa-esm-specifier-transform.ts

```diff
@@ -0,0 +1,57 @@
+import getStream from 'get-stream';
+import isStream from 'is-stream';
+import * as Koa from 'koa';
+import {transformHTMLString} from './transform-html';
+import {transformJavaScriptModuleString} from './transform-javascript-module';
+
+export type TransformSpecifierFunction = (baseURL: string, specifier: string) =>
+    string;
+
+export default middleware;
+export function middleware(transformSpecifier: TransformSpecifierFunction):
+    Koa.Middleware {
+  return async (ctx: Koa.Context, next: Function) => {
+    await next()
```

Comment:
Where'd all the semicolons go? :) What lint config are you using?

Reply:
Was using no lint.  Added tslint.